micro-rdk/src/common/app_client.rs

```diff
@@ -321,7 +321,7 @@ impl AppClient {
                     })
                     .inspect(|_| {
                         log::info!(
-                            "time of day has been set to to {}",
+                            "time of day has been set by config metadata to {}",
```

Comment:
unless there is a strong need there is no point of keep this path for time sync. The question is , when internet is restored is the time sync mostly instantaneous (less than a few seconds)